src/main/resources/application.yml

```diff
@@ -3,12 +3,26 @@ spring:
     url: jdbc:postgresql://localhost:5432/tiny_task
     username: tiny_task
     password: demo123
+  servlet:
+    multipart:
+      max-file-size: 15MB
+      max-request-size: 15MB
 
   jpa:
     database-platform: org.hibernate.dialect.PostgreSQL95Dialect
     hibernate:
       ddl-auto: validate
+    properties:
+      hibernate:
+        temp:
+          use_jdbc_metadata_defaults: false
 
 logging:
   level:
     com.coyoapp.tinytask: DEBUG
+
```

Comment:
backend does not start out of the box due to /storage/.. is read-only. fixed it by create directory /tmp/tiny-task on local machine. an adjustment of the README that the path must be readable or changed would be great

Reply:
Did not pay attention to that, tried to make less config dependent solution.